cmake/Modules/FindKIM-API.cmake

```diff
@@ -54,6 +74,12 @@ if(KIM-API_FOUND)
   pkg_get_variable(KIM-API_CMAKE_Fortran_COMPILER libkim-api CMAKE_Fortran_COMPILER)
 endif()
 
+if(KIM-API_INCLUDEDIR)
+  _KIMAPI_GET_VERSION(KIM-API_VERSION ${KIM-API_INCLUDEDIR}/KIM_Version.h)
+else()
+  set(KIM-API_VERSION 0)
```

Comment:
This results in `DOWNLOAD_KIM=ON` since the find_package() fails.  Probably this is not what we want.  Maybe this should just call `message(FATAL_ERROR ...)`

Reply:
Sorry, could you describe the issue a bit more?

Reply:
If find_package(KIM-API QUIET) does not succeed, then the `DOWNLOAD_KIM` variable is set to `ON`:
https://github.com/lammps/lammps/blob/8cfdf4fad5937e3d7ca1aa867869fd32cbdcd23c/cmake/Modules/Packages/KIM.cmake#L8-L16

Reply:
So we should just not set it to `0` in the `else` case?

Reply:
Assuming we want to keep the "auto promote to `DOWNLOAD_KIM=ON`" feature (I'm not fully convinced it is a good thing), then I think we need to change line 80 to a `message(FATAL_ERROR ...)` instead of `set(KIM-API_VERSION 0)`

Reply:
Done.